installation/README.md

```diff
@@ -1,204 +1,121 @@
 # Installation
 
-At EDA you'll be using three tools every day: a terminal (the 'command line'), an editor (Atom), and Git. You'll also need a web browser of course, and for consistency we suggest you use Google Chrome.
 
-Many EDA staff also like using Zsh - the Z shell - and usually a framework for it called [Oh My Zsh](https://github.com/robbyrussell/oh-my-zsh). To be clear, you don't need to use this to complete the course work but you might like to try it to see if it's a tool you'll enjoy working with.
+At EDA you'll be using three tools every day: a terminal, an editor and Git. You'll also need a web browser of course, and for consistency we suggest you use Google Chrome.
+
+## Installs
+
+1. iTerm2 (Mac) or Babun(Windows)
+2. Homebrew (Mac only)
+3. Git
+4. VS code
+
+<br>
+---------------------------------------
+<br>
 
 
 ## OS X (Mac) instructions
 
-### Terminal
+### iTerm and Homebrew
 
-We recommend you use the [iTerm2](https://www.iterm2.com/) terminal. Although you _could_ complete Phase 0 without it, it's extremely popular among OS X developers, has a lot of extra features and makes things look nicer in the default configuration. Download and install it from the linked site. There's a terminal called, oddly enough, 'terminal' provided with OSX, but iTerm2 is better.
+1. Navigate to [iTerm2](https://www.iterm2.com/)
```

Comment:
We really can leave out the iTerm installation. The installation isn't hard, but Mac OS already has a capabile terminal installed by default. If we're trying to remove all the noise, I think that includes iTerm.